### DISCUSSION TITLE
Console login fails

### DISCUSSION THREAD

**Discussion by LWHaysUSDC** [Security-Onion-Solutions/securityonion#4718] [Unsupported Versions] [answered]
Hi all:

I'm creating a new SO 2.3.52 distributed grid and I have the manager, a search node, and my first sensor node up and phoning home.  salt \* so.status shows all OK

I've run so-allow to permit my desktop IP to access the SO web console, and I've run so-user several times to add/update/enable several usernames.

When I try to login I just get the login screen back.   No error listed on the login screen fwiw.
[kratos-redacted.log](https://github.com/Security-Onion-Solutions/securityonion/files/6772897/kratos-redacted.log)

I've looked through the log files for my username(s) and the only relevant entries I can find are in /opt/so/log/kratos/kratos.log (attached - I redacted my workstation IP address and replaced it with <MY_WS_IP> just so I don't have to ask permission to post.

I previously installed an Eval node (with version 2.3.51, and I can log into that one fine.

Thanks in advance for any help fixing or even debugging this.

Larry

**Comment by jertel** [comment, marked as answer]
The log states that it's not receiving a valid CSRF token. Have you tried from another computer to rule out browser cache or cookie conflicts?

**Reply by LWHaysUSDC** [reply]
Thanks jertel - Chrome/chromium fails but firefox succeeds.   Definitely browser related.

**Reply by jertel** [reply]
Let's keep this discussion separate from #4682 since you're experiencing a login problem in 2.3.52, yet the other discussion encountered a problem after upgrading to 2.3.60, which has a newer version of the authentication software.

I reviewed your HAR logs but it looks like they don't include the POST request and response. In other words, it contains the requests for loading the login page only, not the part when you submitted the login credentials. Also, both F1.har and F2.har are identical - not sure if this was intended.

**Reply by LWHaysUSDC** [reply]
@jertel - OK here is fine.   This is the first time I've tried to capture any browser network logs - I probably screwed it up - I'll try again.   Thanks

**Reply by LWHaysUSDC** [reply]
@jertel I tried again with hopefully better options/procedure.  LMK how it looks.   Thanks
[browserdebug.txt](https://github.com/Security-Onion-Solutions/securityonion/files/6810294/browserdebug.txt)

**Reply by jertel** [reply]
The new HAR log  confirms a cookie conflict. It looks like you chose a mixed case domain or hostname for the manager `SOManager` during installation. Chrome is more strict compared to Firefox and does not allow cookies assigned to `SOManager` to be transmitted to a website with a URL of `https://somanager`. You can either re-install your grid, or you can attempt to update the `url_base` in the global pillar and apply highstate. I'm not sure what is the better approach for your situation but I think you should have enough information to take it from here.

**Reply by LWHaysUSDC** [reply]
Thanks Jason @jertel - Much appreciated

**Reply by LWHaysUSDC** [reply]
FWIW, this is now fixed for me, here's what it took:

I went through and made all my hostnames lowecase and rebooted everything
I tried re-running "sudo SecurityOnion/setup/so-setup iso", which succeeded on the manager but then failed on the rest of the grid with certificate errors - "failed to pull somanager.... x509: issuer name does not match subject from issuing certificate"
At this point I deleted my manager VM, rebuilt it from scratch (with lowercase hostname), re-ran setup, succeeded
reran sudo SecurityOnion/setup/so-setup iso on my search and forward nodes successfully (didn't have to start over with centos install)
Now salt \* so.status shows all OK and I can login with chrome.
